src/components/Modals/FiatRamps/views/Manager.tsx

```diff
@@ -82,9 +82,15 @@ const ManagerRouter: React.FC<ManagerRouterProps> = ({ fiatRampProvider }) => {
       moduleLogger.trace({ fn: 'getAddress' }, 'Getting Addresses...')
       const payload = { wallet }
       try {
-        supportsETH(wallet) && setEthAddress(await ethereumChainAdapter.getAddress(payload))
-        supportsBTC(wallet) && setBtcAddress(await bitcoinChainAdapter.getAddress(payload))
-        supportsCosmos(wallet) && setCosmosAddress(await cosmosChainAdapter.getAddress(payload))
+        supportsETH(wallet) &&
+          ethereumChainAdapter &&
+          setEthAddress(await ethereumChainAdapter.getAddress(payload))
+        supportsBTC(wallet) &&
+          bitcoinChainAdapter &&
+          setBtcAddress(await bitcoinChainAdapter.getAddress(payload))
+        supportsCosmos(wallet) &&
+          cosmosChainAdapter &&
+          setCosmosAddress(await cosmosChainAdapter.getAddress(payload))
```

Comment:
pref for readability
```suggestion
        if (supportsETH(wallet) && ethereumChainAdapter) {
          setEthAddress(await ethereumChainAdapter.getAddress(payload))
        }
        if (supportsBTC(wallet) && bitcoinChainAdapter) {
          setBtcAddress(await bitcoinChainAdapter.getAddress(payload))
        }
        if (supportsCosmos(wallet) && cosmosChainAdapter) {
          setCosmosAddress(await cosmosChainAdapter.getAddress(payload))
        }
```